tests/10-migration-backend-tests/verification/expected/media-file.json

```diff
@@ -1,17 +1,17 @@
 {
   "type": "media",
   "bundle": "file",
-  "name": "Ilford Temperature Compensation Chart",
```

Comment:
Is there any particular reason for modifying the contents of 10-migration-backend-tests and 11-file-deletion-tests?  It's not immediately obvious how it's related to this PR at first glance.. not that it needs to be reverted or anything.